openlr/openlr_tests/decoded_path_test.cpp

```diff
@@ -22,7 +22,7 @@
 #include <sstream>
 #include <vector>
 
-#include "3party/pugixml/pugixml/src/pugixml.hpp"
+#include <pugixml.hpp>
 #include "3party/pugixml/utils.hpp"
```

Comment:
Please check and fix this one and other similar includes (if any).

Reply:
Could you give specifics, about what you think should to be fixed?
This `utils.hpp` actually does not seem to be part of pugixml, and not even a 3party.
This is the only file which includes it, and it is also unclear why, as apparently none of the functions (neither `XMLToString` nor `DebugPrint` seems to be referenced).
Should it be perhaps moved out of `3party` or if indeed not needed, then perhaps removed?

Reply:
Please remove it if it is not used.

Reply:
Removed.